app/controllers/application_controller.rb

```diff
@@ -753,7 +753,7 @@ def get_reports_menu(group = current_group, tree_type = "reports", mode = "menu"
     user = current_user
     @sb[:grp_title] = reports_group_title
     @data = []
-    if (!group.settings || !group.settings[:report_menus] || group.settings[:report_menus].blank?) || mode == "default"
+    if (!group.settings || group.settings[:report_menus].blank?) || mode == "default"
```

Comment:
I know I can further simplify using using `.try(:[], ...)`.

I'll try to explain myself, why I didn't do it in the first place.

When doing pr like this, I
 * look-up the whole codebase for the same abuse, using various patterns 
 * fix one type of abuse in time and move to the next
 * try to make single change consistently to make the reviewers life easier (and to avoid styling discussions)
 * introducing things like: `.try(:[], ...)` would require me to restart the cycle from top.

@bdunne If you think the change to use `.try(:[], ...) would be welcomed&merged, I volunteer to do it in the whole codebase. Please consider whether this change would be welcomed, as it might be considered not easy to read by some people.